store/mongo/datastore_mongo_test.go

```diff
@@ -1680,6 +1679,179 @@ func TestMongoDeleteTokensByUserId(t *testing.T) {
 	}
 }
 
+func TestMongoDeleteTokensByUserIdExceptCurrentOne(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping in short mode.")
+	}
+
+	testCases := map[string]struct {
+		tenant   string
+		user     string
+		inTokens []interface{}
+
+		outTokens []jwt.Token
+		outError  string
+	}{
+		"ok": {
+			user: oid.NewUUIDv5("user-1").String(),
+			inTokens: []interface{}{
+				jwt.Token{
+					Claims: jwt.Claims{
+						ID:       oid.NewUUIDv5("id-1"),
+						Subject:  oid.NewUUIDv5("user-1"),
+						Audience: "audience",
+						ExpiresAt: jwt.Time{
+							Time: time.Now().
+								Add(time.Hour),
+						},
+						IssuedAt: jwt.Time{
+							Time: time.Now(),
+						},
+						Issuer: "iss-1",
+						NotBefore: jwt.Time{
+							Time: time.Unix(7890, 0),
+						},
+						Scope: "scope-1",
+						User:  true,
+					},
+				},
+				jwt.Token{
+					Claims: jwt.Claims{
+						ID:       oid.NewUUIDv5("id-2"),
+						Subject:  oid.NewUUIDv5("user-1"),
+						Audience: "audience",
+						ExpiresAt: jwt.Time{
+							Time: time.Now().
+								Add(time.Hour),
+						},
+						IssuedAt: jwt.Time{
+							Time: time.Now(),
+						},
+						Issuer: "iss-1",
+						NotBefore: jwt.Time{
+							Time: time.Unix(7890, 0),
+						},
+						Scope: "scope-1",
+						User:  true,
+					},
+				},
+				jwt.Token{
+					Claims: jwt.Claims{
+						ID:       oid.NewUUIDv5("id-3"),
+						Subject:  oid.NewUUIDv5("user-2"),
+						Audience: "audience",
+						ExpiresAt: jwt.Time{
+							Time: time.Now().
+								Add(time.Hour),
+						},
+						IssuedAt: jwt.Time{
+							Time: time.Now(),
+						},
+						Issuer: "iss-1",
+						NotBefore: jwt.Time{
+							Time: time.Unix(7890, 0),
+						},
+						Scope: "scope-1",
+						User:  true,
+					},
+				},
+			},
+			outTokens: []jwt.Token{
+				{
+					Claims: jwt.Claims{
+						ID:       oid.NewUUIDv5("id-1"),
+						Subject:  oid.NewUUIDv5("user-1"),
+						Audience: "audience",
+						ExpiresAt: jwt.Time{
+							Time: time.Now().
+								Add(time.Hour),
+						},
+						IssuedAt: jwt.Time{
+							Time: time.Now(),
+						},
+						Issuer: "iss-1",
+						NotBefore: jwt.Time{
+							Time: time.Unix(7890, 0),
+						},
+						Scope: "scope-1",
+						User:  true,
+					},
+				},
+				{
+					Claims: jwt.Claims{
+						ID:       oid.NewUUIDv5("id-3"),
+						Subject:  oid.NewUUIDv5("user-2"),
+						Audience: "audience",
+						ExpiresAt: jwt.Time{
+							Time: time.Now().
+								Add(time.Hour),
+						},
+						IssuedAt: jwt.Time{
+							Time: time.Now(),
+						},
+						Issuer: "iss-1",
+						NotBefore: jwt.Time{
+							Time: time.Unix(7890, 0),
+						},
+						Scope: "scope-1",
+						User:  true,
+					},
+				},
+			},
+		},
+	}
+
+	for name, tc := range testCases {
+		t.Logf("test case: %s", name)
```

Comment:
Can you please replace this with `t.Run(name, func(t *testing.T)` ?

Reply:
Done, for all the tests.